docs/infrastructure.md

```diff
@@ -0,0 +1,155 @@
+# Setting up infrastructure for using fondant
+
+## Table of Contents
+1. [Introduction](#introduction)
+2. [If you already have a kubernetes cluster](#If-you-already-have-a-kubernetes-luster)
+3. [Fondant on AWS](#fondant-on-aws)
+4. [Fondant on Google Cloud](#fondant-on-google-cloud)
+5. [Fondant on a local machine Using Minikube](#fondant-on-a-local-machine-using-minikube)
+
+## Introduction
+Fondant is built on top of [Kubeflow Pipelines](https://www.kubeflow.org/docs/components/pipelines/v1/introduction/), so to run it we'll need:
+
+- A kubernetes cluster
+- Kubeflow pipelines installed on the cluster
+- A registry to store custom component images like (docker hub, Github Container Registry, etc)
+
+This can be on any kubernetes cluster, if you don't have access to a setup like this or you feel uncomfortable to setup your own we have provided some basic scripts to get you started on GCP or on a small scale locally.
+
+**IMPORTANT: These script serve just a kickstart to help you start using fondant fast, these are not production ready environments.**
```

Comment:
Would be nice to use [Admonitions](https://squidfunk.github.io/mkdocs-material/reference/admonitions/) here.

Reply:
Thx! I'll take a look